packages/nouns-webapp/src/components/AuctionPriceRange/AuctionPriceRange.tsx

```diff
@@ -0,0 +1,74 @@
+import { BigNumber as BigNumberEthers, utils } from 'ethers';
+import BigNumber from 'bignumber.js';
+import TruncatedAmount from '../TruncatedAmount';
+
+interface Props {
+  currentPrice: BigNumberEthers;
+  targetPrice: BigNumberEthers;
+  reservePrice: BigNumberEthers;
+}
+
+export const AuctionPriceRange = (props: Props) => {
+  const { currentPrice, targetPrice, reservePrice } = props;
+
+  const numPriceBlocks = blockColors.length;
+  const maxPrice = targetPrice.mul(2);
+
+  const currentPricePercentage =
+    parseFloat(utils.formatEther(currentPrice)) / parseFloat(utils.formatEther(maxPrice));
+
+  const activeIndex = Math.floor(currentPricePercentage * numPriceBlocks);
+
+  return (
+    <div className="w-full">
+      <div className="flex items-center justify-between w-full space-x-1">
+        {blockColors.map((color, index) => (
+          <PriceBlock color={color} key={index} isActive={index === activeIndex} />
+        ))}
+      </div>
+      <div className="mt-2.5 flex justify-between">
+        <h4 className="font-medium">
+          <TruncatedAmount amount={BigNumber(reservePrice.toString() || '0.00')} />
+        </h4>
+        <h4 className="font-medium">
+          <TruncatedAmount amount={BigNumber(targetPrice.toString() || '0.00')} />
+        </h4>
+        <h4 className="font-medium">
+          <TruncatedAmount amount={BigNumber(maxPrice.toString() || '0.00')} />
```

Comment:
On test contracts these values may look off, they come from the contract config.